css-borders-4/Overview.bs

```diff
@@ -336,29 +336,171 @@ The 'border-radius' shorthand</h4>
 	See [[CSS3BG]].
 
 <h3 id=corner-shaping>
-Corner Shaping: the 'corner-shape' property</h3>
+Corner Shaping</h3>
+
+By default, non-zero border-radii define a quarter-ellipse that rounds the affected corners.
+However in some cases, other corner shapes are desired.
+The 'corner-shape' property group specifies a reinterpretation of the radii to define other corner shapes.
+
+The different shapes applicable to 'corner-shape' can be expressed as different parameters to a superellipse.
+A superellipse is a closed curve resembling an ellipse, and based on its `k` parameter can express all the shapes between a square, an ellipse, and a notch.
+
+To allow full expression as well as interpolation, the 'corner-shape' properties can provide its own superellipse parameter using the 'superellipse()' function,
+or use one of the supplied keywords which represent commonly used parameters. See the <<corner-shape-value>> definition for details.
+
+<h4 id=corner-shape-rendering>
+Rendering 'corner-shape'</h4>
+'corner-shape' works alongside 'border-radius', and does not have any visual effect with a 'border-radius' of 0.
+It acts as an alteration on top of the default round 'border-radius', and thus can be used as progressive enhancement.
+
+Like 'border-radius', 'corner-shape' clips elements according to the [=overflow=] rules, and applies to the rendering of the border.
+Since stroking a superellipse accurately may be computationally intensive, user agents may approximate the path using bezier curves,
+as well as account for sharp edges and overlaps.
+
+Issue: describe the rendering requirements in more detail.
+
+<h4 id=corner-shape-value>
+'corner-shape' values</h4>
+
+	<pre class=prod>
+		<dfn><<corner-shape-value>></dfn> = ''round'' | ''scoop'' | ''bevel'' | ''notch'' | ''straight'' | ''squircle'' | superellipse(<<number [0,&infin;]>>)
+	</pre>
+
+	<dl dfn-type="value" dfn-for="corner-shape-value">
+		<dt><dfn>''round''</dfn>
```

Comment:
does it say somewhere that this is the default? 

Also, is it worth mentioning that this is the "current behavior", i.e., rounded corners?

Reply:
It does, see the different properties